pkg/api/customization/kontainerdriver/formatter.go

```diff
@@ -26,15 +26,27 @@ func NewFormatter(manangement *config.ScaledContext) types.Formatter {
 	return format.Formatter
 }
 
-const clusterByKontainerDriverKey = "clusterbyKontainerDriver"
+const clusterByGenericEngineConfigKey = "genericEngineConfig"
 
-func clusterByKontainerDriver(obj interface{}) ([]string, error) {
+// IndexFunction that uses cluster's genericEngineConfig map[string]interface{}
```

Comment:
This should follow go formatting for [comments](https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences). Don't add the type to the comment.